CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ Highlights
 Bug-fixes
 ---------
 
+* Neither of the `record` constructors `_,_` in `Algebra.Definitions.RawMagma`
+  nor `Relation.Binary.Construct.Interior.Symmetric`, introduced to mimic that
+  of `Data.Product.Base`, were introduced with the appropriate fixity, namely
+  `infixr 4 _,_`.
```

Comment:
This is very minor, but can this be written as describing an active fix. At the moment, it technically doesn't actually tell the reader what happened! Something like:

> The `record` constructors `_,_` in `Algebra.Definitions.RawMagma` and `Relation.Binary.Construct.Interior.Symmetric` incorrectly had no assigned fixity. They have been given the fixity of `infixr 4 _,_`?

Reply:
Have used the merge conflict to tidy up the `CHANGELOG`, following your suggestion. Thanks!